src/Resources/Locales/en.json

```diff
@@ -927,6 +927,11 @@
             "unhandledExceptionMessage": "Something unexpected happened. Please try again.",
             "versionDocumentationLink": "Learn more about DTDL version 3"
         },
+        "Errors": {
```

Comment:
do we start with upper case in translation keys?

Reply:
At least in OAT my pattern has been that each section block is caps and the actual key is lowercase.